docs/getting-started/what-is-dotnet.md

```diff
@@ -133,18 +139,17 @@ The .NET Framework on the Windows desktop allows you to create console applicati
 
 - Console applications can be compiled to run cross-platform on Windows, the Macintosh, and Linux.
```

Comment:
"Identical cross-platform code can be compiled into console apps that run on Windows, Mac OS, and Linux."